src/screens/SendSheet.js

```diff
@@ -455,12 +454,7 @@ export default function SendSheet(props) {
   ]);
 
   return (
-    <Container
-      additionalTopPadding
-      as={android && SlackSheet}
-      contentHeight={sheetHeight}
-    >
-      {ios && <StatusBar barStyle="light-content" />}
+    <Container>
```

Comment:
@osdnk are u ok with this change?

Reply:
you both solved in different ways so when I pulled I had 2 handles, this seemed like the best solution and I tested on both OS

Reply:
status bar removal was actually an accident, above comment refers to your other

Reply:
Are you sure it works?